brightcove/brightcove-popup/src/components/modal.tsx

```diff
@@ -82,7 +82,7 @@ const Modal: React.FC<ModelProps> = (props) => {
 
```

Comment:
You already destructure `props` at the start of the component then why `props.selectedVideos` ? Please update it everywhere.